src/index.js

```diff
@@ -118,14 +118,15 @@ function stripLoc(doc, removeLocAtThisLevel) {
   return doc;
 }
 
+var experimentalFragmentVariables = false;
 function parseDocument(doc) {
   var cacheKey = normalize(doc);
 
   if (docCache[cacheKey]) {
     return docCache[cacheKey];
   }
 
-  var parsed = parse(doc);
+  var parsed = parse(doc, { experimentalFragmentVariables });
```

Comment:
unfortunately we're not compiling this file just yet, so im going to make this an explicit object